### PR TITLE
Local DB Manager

### PR DESCRIPTION
**Pull request by olivecylinder**

### CODE REVIEW

**Comment by olivecylinder** [comment]
[מנהל DB באנדרואיד לתחנות](https://trello.com/c/YiCjPRFy/19-%D7%9E%D7%A0%D7%94%D7%9C-db-%D7%91%D7%90%D7%A0%D7%93%D7%A8%D7%95%D7%90%D7%99%D7%93-%D7%9C%D7%AA%D7%97%D7%A0%D7%95%D7%AA)

File: Nahagos/app/src/main/java/com/nahagos/nahagos/db/Tables.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+import org.jetbrains.annotations.NotNull;
+
+public class Tables {
+    @Entity
+    public static class Stops {
```
[the remark this answers is not in the document]
Comment (olivecylinder) [reply Nahagos/app/src/main/java/com/nahagos/nahagos/db/Tables.java:11]: The `Stops` class represents the stops tables defined in gtfs.

[the remark this answers is not in the document]
Comment (olivecylinder) [reply Nahagos/app/src/main/java/com/nahagos/nahagos/db/Tables.java:11]: regarding indexes, decided to drop it, couldn't find enough info regarding indexes with room combined with loading from assets.

**Review by goatOfLife — APPROVED** [review]
Either fixed what I've said, or told me why it should be that way reasonably. very good.

**Pull request merged**